src/components/Book.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Book({
+  category,
+  title,
+  author,
+  onDelete,
+  isCompleted,
+  toggleStatus,
+}) {
+  return (
+    <div className={`book-list ${isCompleted ? 'completed' : ''}`}>
+      <div className="bookcont">
+        <div className="books-info">
+          <div className="book-cata">{category}</div>
+          <p className="book-title">{title}</p>
+          <p className="book-author">{author}</p>
+          <span className="btn" id="commit">
+            Comments
+          </span>
+          <button type="button" className="btn" id="delete" onClick={onDelete}>
+            Remove
+          </button>
+          <span className="btn" id="edit">
+            Edit
+          </span>
```

Comment:
- [ ] Please match to original design and ensure to add to dividers as I pointed it out on the screenshot on top of this comment.